lib/linguist/languages.yml

```diff
@@ -5066,6 +5066,15 @@ Shen:
   tm_scope: source.shen
   ace_mode: text
   language_id: 348
+Sieve:
+  type: programming
+  tm_scope: none
```

Comment:
1.	Change this to:
	~~~diff
	 Sieve:
	   type: programming
	-  tm_scope: none
	+  tm_scope: source.sieve
	~~~

2.	Then update the submodule containing your spiffy new Sieve grammar:

	~~~sh
	(cd ./vendor/grammars/language-etc && git pull)
	script/list-grammars
	git add --all .
	~~~

_(**NB:** These instructions are informative rather than literal)_